src/components/RichTextEditor/serializer/serializeToHtml.spec.ts

```diff
@@ -34,6 +34,21 @@ describe('serializeNodesToHtml()', () => {
         expect(serialized).to.contain('column-gap:normal');
     });
 
+    it('should serialize the html text', () => {
+        const node = [
+            {
+                type: 'p',
+                children: [
+                    {
+                        text: "<img src=x onerror='alert(123)'/>\\n",
+                    },
+                ],
+            },
+        ];
+        const serialized = serializeNodesToHtml(node);
+        expect(serialized).to.be.a('string');
```

Comment:
Wouldn't it be good to also check if the returned string is as expected, instead of only the type of it? Like on the `serializes an empty line to a linebreak` test case?